src/main/scala/cromwell/util/google/GoogleCloudStorage.scala

```diff
@@ -36,6 +36,15 @@ case class GoogleCloudStorage(client: Storage) {
     new GcsBucketInfo(bucketName, bucket.getLocation, bucket.getTimeCreated, bucket.getOwner)
   }
 
+  /**
+    * Gets a CRC code from a GCS object. This is a CRC32c checksum which can serve as a hash. It is resistant to
```

Comment:
I think it's more "robust" than "resistant"; it doesn't stop you from doing composite uploads.  :stuck_out_tongue: